App/Screens/Auth/Register.js

```diff
@@ -90,7 +91,8 @@ class Register extends Component<Props, State> {
       try {
         const {data} = await controller.login(email, password)
         await controller.loadAndStoreUser(data)
-        this.navigate('Home')
+        const user = await UserController.get()
```

Comment:
`UserController.get()` is not asynchronous.
```suggestion
        const user = UserController.get()
```